src/wavm/codegen.cpp

```diff
@@ -6,13 +6,17 @@
 #include <WAVM/WASM/WASM.h>
 #include <WAVM/WASTParse/WASTParse.h>
 
+#include <faabric/util/logging.h>
 #include <faabric/util/files.h>
 
 using namespace WAVM;
 
 namespace wasm {
-std::vector<uint8_t> wavmCodegen(std::vector<uint8_t>& bytes)
+std::vector<uint8_t> wavmCodegen(std::vector<uint8_t>& bytes, 
```

Comment:
So these changes to `doCodegen` and `wavmCodegen` are for better error reporting I guess? It's alright, just a bit confusing that they are included in this PR.

Reply:
I guess it depends on how strict we want to be with single purpose PRs. Given how long the build takes at the moment I think it's OK to include small improvements like this alongside other changes.

Reply:
I take your point that the title is kind of misleading. Although it's updating the submodules that contain changes related to Horovod/ MXNet, the changes in this repo are more generally relevant to building libraries. I've renamed the PR to better reflect the fact that it's a bit of a catch-all